### PR TITLE
Image puzzle game -  PuzzlePage and PuzzlePageContent components

### PR DESCRIPTION
**Pull request by majdajroudi**
In this PR, I added the page for the game, _PuzzlePage_, and the space which is the content of the page, _PuzzlePageContent_. the page of the puzzle has its own route in the App.jsx and puzzleSpace is imported in the page's container. 

### Game Description:
The game is a simple image puzzle game with multiple images to be solved, the tiles of the puzzle are ordered vertically where each row is one tile (as shown in the image attached). The user can choose which image they want to solve. The content of the page will be a card divided into two sections, upper part is list of images and bottom part is the current image the user is solving. The first image in the list is the one in the current image space by default, when the user solves the puzzle the image that was solved is displayed as a whole.

### PR Summary: 
- Add PuzzlePage container 
- Add ~~PuzzleSpace~~ PuzzlePageContent component
- Add route for the the page of the game

### CODE REVIEW

**Review by louisrli — COMMENTED** [review]
i recommended some renaming on the other PR, otherwise this LGTM. rename and i'll approve

File: src/components/PuzzleSpace/index.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Layout, Card } from "antd";
+import "./index.css";
+
+const GameSpace = () => {
```
Comment (ssinikoglu) [inline src/components/PuzzleSpace/index.jsx]: Since there is single function in the component, you may rename GameSpace as PuzzleSpace.

**Review by ssinikoglu — APPROVED** [review]
Your code is clean and I liked your detailed explanation in the PR.

**Comment by majdajroudi** [comment]
Updated this PR's name and the other PR's name. I also renamed the components, functions and class names as advised. PTAL

**Review by louisrli — APPROVED** [review]

**Comment by louisrli** [comment]
lgtm but conflicts

**Comment by majdajroudi** [comment]
Fixed conflicts and PR approved, PTAL @Ammar-64

**Pull request merged**